### DISCUSSION TITLE
Any shortcut to get first matching record by a field?

### DISCUSSION THREAD

**Discussion by mon-jai** [nushell/nushell#5823] [Q&A] [answered]
In the line above, is there any shortcut to exteact the first matching record?
e.g. something like `get-first name == PROMPT_COMMAND`

```
env | where name == PROMPT_COMMAND | get 0
```

**Comment by fdncred** [comment, marked as answer]
I don't understand what you're asking. `env | where name == PROMPT_COMMAND` gets the only matching env var with that name. Are you saying you want to look at the block contained in `PROMPT_COMMAND`?

**Reply by mon-jai** [reply]
> Are you saying you want to look at the block contained in PROMPT_COMMAND?

Yap. Any way to get the first matching result?

**Reply by fdncred** [reply]
This is as close as you can get right now. `view-source $env.PROMPT_COMMAND`

**Comment by kubouch** [comment]
Not directly, you'd need to make your own wrapper command.

For environment variables, you can do just `$env.PROMPT_COMMAND`.
